purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -130,6 +130,17 @@ class Purchases internal constructor(
         purchasesOrchestrator.removeUpdatedCustomerInfoListener()
     }
 
+    /**
+     * Google Play only, no-op for Amazon.
+     * If the user has had a payment declined, this will show a toast notification notifying them and
+     * providing instructions for recovery of the subscription.
+     *
+     * https://developer.android.com/google/play/billing/subscriptions#in-app-messaging
```

Comment:
we should replace this with a rev.cat link that we own

Reply:
Done